packages/serverless-contracts/src/contracts/apiGateway/testUtils/__tests__/mockHandlerInput.test.ts

```diff
@@ -91,7 +97,12 @@ describe('apiGateway lambda handler mock input', () => {
 
   it('should generete event with some defined keys', () => {
     const [event] = getMockHandlerInput(httpApiContract, {
-      body: { foo: 'azer', bar: 123 },
+      body: {
+        foo: 'azer',
+        bar: 123,
+        baz: undefined,
```

Comment:
is it working if we don't specify `baz` or it should be explicitly undefined ?

Reply:
If you don't especially specify baz, a random value will be generated